nbextensions/bulkImportCell/tabs/xsvGenerator.js

```diff
@@ -208,7 +426,8 @@ define(['StagingServiceClient', 'common/html', 'common/runtime', 'common/ui', 'u
                 });
             }
 
-            const s_have = Object.keys(resp.files_created).length === 1 ? ' has' : 's have';
+            const newFiles = new Set(Object.values(resp.files_created));
```

Comment:
Use a set here as `Object.values(resp.files_created)` may contain copies of the same string if an Excel file is created.